server/services/save-message.js

```diff
@@ -0,0 +1,20 @@
+const { Message } = require('../models/messages');
+require('dotenv').config();
+
+async function saveMessage(message, username, room, __createdtime__, socketId) {
```

Comment:
just pass this parameters as object, smth like 'prepared data', and then just spread it when you create a Message